prov/efa/src/rdm/efa_rdm_ope.h

```diff
@@ -85,7 +85,7 @@ enum rxr_cuda_copy_method {
 	RXR_CUDA_COPY_LOCALREAD   /** device driven copy by using local RDMA read */
 };
 
-struct rxr_op_entry {
+struct efa_rdm_ope {
```

Comment:
I agree with the rename, but I think we are losing a little bit of good naming here.  Can we put a comment around this struct definition that "ope" is an "operation entry" just so it's documented somewhere?

Reply:
Added some doxgen string